app/component/console/console.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <form novalidate
+    ng-submit="consoleCtrl.player.move(consoleCtrl.direction)">
+    <input type="text" ng-model="consoleCtrl.direction">
```

Comment:
Some styling here could really be needed. At first, I didn't even see your input bar on your page because it just blended in with everything else.